### ISSUE TITLE
Less Linter: Unrecognised input in

### ISSUE THREAD

**Issue by almereyda** [JedWatson/sydjs-site#86]
As much as I like KeystoneJS and this repository, I also do like working and running software.

```
[almereyda@neocortex sydjs-site] $node_modules/less/bin/lessc -l public/styles/site/type.less
ParseError: Unrecognised input in /home/almereyda/tmp/sydjs-site/public/styles/site/type.less on line 25, column 1:
24  
25 @media (max-width: @screen-md-max) {
26 	h1, .h1 { font-size: 2.8em; }
```

Maybe this search helps a bit digging it down? http://lmgtfy.com/?q=less+unrecognised+input+%40media+max-width

**Comment by JedWatson** [comment]
This is pretty weird - not sure when that broke, but last time I was working with the site (and for the last deployment) it was just fine. There's search hits on the error go back a number of years, which makes it even stranger that it's broken now, but we can reproduce on a fresh install. Anyway, fixed for now by #88 - thanks @Noviny for contributing that.

**Comment by JedWatson** [comment]
Just realised this is the same issue mentioned in keystonejs/keystone#1299. Fix has been added to the less files in this repo too.

Also just a comment @almereyda lmgtfy comes across a bit rude on open source issues, I assume you meant no offence but in this case I wasn't aware this was even breaking (and still don't really understand what caused it to _start_ breaking so recently)

Turns out the underlying issue was related to whitespace before the `@media` query and is a problem with the LESS parser, which isn't something those google results actually helped narrow down.

I'm trying to say this without coming across the wrong way, but this code is freely published with the hope it is helpful to others. I'd recommend in the future either simply reporting an issue, or actually helping identify the cause + fix, but linking to something that treats others like they're idiots is quite abrasive.
